assets/js/blocks/cart-checkout/checkout-i2/inner-blocks/checkout-sample-block/index.tsx

```diff
@@ -39,7 +45,14 @@ registerCheckoutBlock( 'woocommerce/checkout-sample-block', {
 			multiple: true,
 			reusable: false,
 		},
-		attributes: {},
+		attributes: {
+			lock: {
+				type: 'object',
+				default: {
+					remove: true,
+				},
+			},
+		},
```

Comment:
Forcing is currently separated from locking, but if you force a block to render, you should prevent it from being deleted as well. This can be handled at the registration in `registerCheckoutBlock` by automatically adding those attributes.

Reply:
I was actually wrong here @mikejolley Forcing doesn’t depend on a `force: true` attribute anymore ,it depends on having `attributes.lock.remove: true` which would cause the block to be inserted. It reduces the surface area for the API and works as expected. This is only for blocks registered with `registerCheckoutBlock`.